### ISSUE TITLE
Reference

### ISSUE THREAD

**Issue by shijunil** [xuanchien/abstractive-summarizer-python#1]
Hi
Is there some way i can run the code without gurobi?
Also can you please send your reference or research paper.

Thanks
Shiju

**Comment by bitmega** [comment]
Hi, actually this code is a part of my research, and it needs Gurobi to run. I am implementing totally this based on the paper: http://arxiv.org/abs/1506.01597. In fact I am still making constant changes to this code :)

**Comment by xuanchien** [comment]
Hi, actually this code is a part of my research, and it needs Gurobi to run. I am implementing totally this based on the paper: http://arxiv.org/abs/1506.01597. In fact I am still making constant changes to this code :)

**Comment by shijunil** [comment]
thank you. I m also working in the field of abstractive text summarization
 for my research and trying new codes/ methods. I m finding problem in
getting the license key for Gurobi. Any suggestions?. Also is there any
similar code you are aware of.
Thanks
Shiju

**Comment by xuanchien** [comment]
Gurobi provides free academic license, you can apply for the license directly on their website and download the license key.

**Comment by shijunil** [comment]
Yes I tried doing that but there seems to be some error with  respect to IP address. My institution IP address in not there in the list of Academic domain. I have written a mail to them. Thanks
